_pages/install/integrations/opf-integration/opf-child-pages/1-open-payment-framework-in-spartacus.md

```diff
@@ -29,64 +29,134 @@ For more information about `--opfBaseUrl` and `--commerceCloudPublicKey`, see [C
 
 Open payment framework is CMS-driven. If you are using the [Spartacus Sample Data Extension](link), the open payment framework CMS components are already enabled. However, if you decide not to use the `spartacussampledata` extension, you can enable the open payment framework CMS components manually through ImpEx.
 
-### Adding the CMS Components Manually Using ImpEx
+### Understanding the CMS Structure
 
-To add all of the necessary CMS components and related data for open payment framework, import the following ImpEx:
+OPF requires three CMS elements to work properly: content pages, content slots, and page-slot relations. If you are not using the Spartacus Sample Data Extension, you can add these elements manually step-by-step. The following sections show you how to do this.
+
+Content pages define the checkout pages users navigate to, content slots are areas where components are placed within pages, and page-slot relations connect pages to slots and define the layout structure. Each script is prepended with the following variables:
+
+```text
+$contentCatalog=electronics-spaContentCatalog
+$contentCV=catalogVersion(CatalogVersion.catalog(Catalog.id[default=$contentCatalog]),CatalogVersion.version[default=Online])[default=$contentCatalog:Online]
+```
+
+The `$contentCatalog` defines which content catalog to work with, and `$contentCV` (content catalog version) specifies the version of that catalog. These variables ensure that all the OPF components, pages, and slots are created in the correct catalog and version. You need to customize these variables to match your specific content catalog.
+
+#### Creating Content Pages
+
+To implementing the OPF components in your Spartacus storefront, you first create the actual checkout page that users navigate to during the OPF flow. This page is accessible through a specific URL, such as `/checkout/opf-payment-and-review`, and uses the standard Spartacus checkout template structure. The page definition includes the template type, URL routing, and approval status that Spartacus needs to properly render the checkout experience.
```

Comment:
"This page is accessible through a specific URL..."

Should this say "route" instead of "URL"? 

@Matejk00

Reply:
Hi @gladius-mtl you're right, I think that route would be a better fit in this case.